Covinator/Covinator/Pages/PfizerVaccination.cshtml

```diff
@@ -102,7 +115,7 @@
                 {
                 <tr>
                     <td>@pfizerVaccineDistributionAllocations.Jurisdiction </td>
-                    <td>@pfizerVaccineDistributionAllocations.WeekOfAllocations of allocations </td>
+                    <td>@pfizerVaccineDistributionAllocations.WeekOfAllocations.DateTime.ToShortDateString() </td>
                     <td>@pfizerVaccineDistributionAllocations.The1StDoseAllocations </td>
```

Comment:
• The table on Pfizer page was taking the time value (time of allocation) which was not required. So, I formatted it to show just the day, month and year of allocation.